pkg/action/executor/rescheduler.go

```diff
@@ -214,12 +214,12 @@ func (r *ReScheduler) reSchedule(action *turboaction.TurboAction) (*turboaction.
 	}
 
 	preScheduler, err := f(r.kubeClient, namespace, parentName, "", DefaultNoneExistSchedulerName)
+	defer f(r.kubeClient, namespace, parentName, DefaultNoneExistSchedulerName, preScheduler)
```

Comment:
or this way may be better:

restore := func() {
		if preScheduler != "" {
			f(client, nameSpace, *noexistSchedulerName, preScheduler)
		}
	}
	defer restore()